src/main/scala/xiangshan/cache/mmu/PageTableWalker.scala

```diff
@@ -858,6 +1095,12 @@ class HPTWIO()(implicit p: Parameters) extends MMUIOBaseBundle with HasPtwConst
     val req = ValidIO(new PMPReqBundle())
     val resp = Flipped(new PMPRespBundle())
   }
+  val bitmap = new Bundle{
+    val req = DecoupledIO(new bitmapReqBundle())
+    val resp = Flipped(DecoupledIO(new bitmapRespBundle()))
+  }
```

Comment:
Should be a option bundle? Then we need not say:
```
  if (HasBitmapCheck) {
    io.bitmap.xxx := xxx
  } else {
    io.bitmap.xxx := DontCare
  }
```

If we don't set these bundles as options, we can only rely on chisel's functionality to optimise them away; not that they won't be generated initially